open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_rolling_window_mean(df, column, window):
</pre><p>Let's change this as <code>get_rolling_window</code> and move the <code>mean()</code> out of the function. So this could be reuased for <code>get_rt</code> below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>DONE</p>